scripts/spack/packages/axom/package.py

```diff
@@ -87,6 +87,7 @@ class Axom(CachedCMakePackage, CudaPackage, ROCmPackage):
 
     variant("examples", default=True, description="Build examples")
     variant("tools", default=True, description="Build tools")
+    variant("tutorials", default=True, description="Build tutorials")
```

Comment:
It's minimal added build time, right? I would think most people using Spack to build Axom just care about getting a library to link into their project.